search_events_app/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import EventListView
```

Comment:
Imports should be grouped into groups. Each group needs to be separated from each other with a blank line.
The groups are, in order:
1. All standard library imports, including backports installed by PyPi (e.g. enum, mock, etc.).
2. Django imports (e.g. _from django.urls import path_)
3. Project imports (e.g. _from .views import EventListView_)